libs/clients/social-insurance-administration/src/lib/utils.ts

```diff
@@ -0,0 +1,53 @@
+import fetch from 'isomorphic-fetch'
+import { logger } from '@island.is/logging'
+
+export const createWrappedFetchWithLogging = (
```

Comment:
You shouldn't require this utility, rather use the `EnhancedFetch` and the logger properties there.